content/en/tracing/guide/metrics_namespace.md

```diff
@@ -21,15 +22,44 @@ aliases:
   - /tracing/getting_further/metrics_namespace
 ---
 
-The trace metrics namespace is `trace.<name>.<metrics>{<tags>}` where
+## Overview
 
-* `<name>`: is the name of the operation i.e. the `span.name`: (*redis.command*, *pylons.request*, *rails.request*, *mysql.query*)
-* `<metrics>` is about the hits, errors or latency ("request.hits", etc.)
-* `<tags>` the metrics are tagged by service & resource
+Tracing application metrics are collected after [enabling trace collection][1] and [instrumenting your application][2]. These metrics are available for dashboards and monitors.
 
-So for pylons it might be `trace.pylons.request.hits{service:web_server}`.
+### Metric names
+
+Trace metric names are formatted as `trace.<NAME>.<METRIC>` or `trace.<NAME>.<METRIC_2ND_PRIM_TAG>`.
+
+| Parameter               | Description                                                                                                             |
+|-------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| `<NAME>`                | The name of the operation or `span.name` (examples: `redis.command`, `pylons.request`, `rails.request`, `mysql.query`). |
+| `<METRIC>`              | The name of the metric (examples: `duration`, `hits`, `span_count`).                                                    |
+| `<METRIC_2ND_PRIM_TAG>` | If the metric name accounts for the second primary tag, this tag is part of the metric name.                            |
+
+### Tags
+
+Trace metrics come with a variety of tags. The possible tags are:
+
+* `env`
+* `service`
+* `resource`
+* `sublayer_type`
+* `sublayer_service`
+* `http.status_code`
+* `http.status_class`
+* Datadog Agent tags (including host tag)
```

Comment:
would have written: 
" Datadog Agent tags (including host tag and second primary tag)"

The second primary tag is configured in the UI from the list of the Datadog Agent tags but both ways to write it can be confusing so it's up to you.